tests/lib/nsxt_test.py

```diff
@@ -0,0 +1,930 @@
+    'source_address',
+    'source_address_exclude',
+    'source_port',
+    'option',
+    'platform',
+    'platform_exclude',
+    'translated',
+    'verbatim',
+}
+
+SUPPORTED_SUB_TOKENS = {
+    'action': {'next', 'accept', 'deny', 'reject', 'reject-with-tcp-rst'},
+    'icmp_type': {
+        'alternate-address',
+        'certification-path-advertisement',
+        'certification-path-solicitation',
+        'conversion-error',
+        'destination-unreachable',
+        'echo-reply',
+        'echo-request',
+        'mobile-redirect',
+        'home-agent-address-discovery-reply',
+        'home-agent-address-discovery-request',
+        'icmp-node-information-query',
+        'icmp-node-information-response',
+        'information-request',
+        'inverse-neighbor-discovery-advertisement',
+        'inverse-neighbor-discovery-solicitation',
+        'mask-reply',
+        'mask-request',
+        'information-reply',
+        'mobile-prefix-advertisement',
+        'mobile-prefix-solicitation',
+        'multicast-listener-done',
+        'multicast-listener-query',
+        'multicast-listener-report',
+        'multicast-router-advertisement',
+        'multicast-router-solicitation',
+        'multicast-router-termination',
+        'neighbor-advertisement',
+        'neighbor-solicit',
+        'packet-too-big',
+        'parameter-problem',
+        'redirect',
+        'redirect-message',
+        'router-advertisement',
+        'router-renumbering',
+        'router-solicit',
+        'router-solicitation',
+        'source-quench',
+        'time-exceeded',
+        'timestamp-reply',
+        'timestamp-request',
+        'unreachable',
+        'version-2-multicast-listener-report',
+    },
+    'option': {'first-fragment', 'established', 'tcp-established', 'initial', 'sample', 'rst', 'is-fragment'}
+}
+
+# Print a info message when a term is set to expire in that many weeks.
+# This is normally passed from command line.
+EXP_INFO = 2
+_PLATFORM = 'nsxt'
+
+
+class TermTest(absltest.TestCase):
+
+  def setUp(self):
+    super().setUp()
+    self.naming = mock.create_autospec(naming.Naming)
+
+  def testInitForinet(self):
+    """Test for Term._init_."""
+    inet_term = nsxt.Term(INET_TERM, 'inet')
+    self.assertEqual(inet_term.af, 4)
+    self.assertEqual(inet_term.filter_type, 'inet')
+
+  def testInitForinet6(self):
+    """Test for Term._init_."""
+    inet6_term = nsxt.Term(INET6_TERM, 'inet6', None, 6)
+    self.assertEqual(inet6_term.af, 6)
+    self.assertEqual(inet6_term.filter_type, 'inet6')
+
+  @unittest.skip("_ServiceToStr was removed")
+  def testServiceToStr(self):
+    """Test for Term._ServiceToStr."""
+
+    proto = 6
+    icmp_types = []
+    dports = [(1024, 65535)]
+    spots = [(123, 123)]
+    nsxt_term = nsxt.Term(INET_TERM, 'inet')
+    service = nsxt_term._ServiceToString(proto, spots, dports, icmp_types)
+    self.assertEqual(service, '<service><protocol>6</protocol><sourcePort>'
```

Comment:
More XML (although you have `@unittest.skip` above; would it not make sense to simply delete the test then, if it'll unconditionally be skipped? Is it correct to skip it?)

Reply:
I believe all XML references were removed with the last commit.